Web.Client/Pages/Electives/SubjectList.razor.cs

```diff
@@ -37,9 +37,18 @@ protected override async Task OnInitializedAsync()
 
 		if ((await ClientAuthService.GetCurrentClaimsPrincipal()).IsInRole(nameof(Role.Student)))
 		{
+			// Get grade (shouldn't be null, user is student)
 			var gradeId = await ClientAuthService.GetCurrentStudentGradeIdAsync();
+			Debug.Assert(gradeId != null, nameof(gradeId) + " != null");
```

Comment:
Pokud bys použil `Contract.Assert<InvalidOperationException>(grade != null)`, tak se ti tam ta message vytvoří sama (ale zas by se ti to spouštělo i v Release buildech.